ephios/settings.py

```diff
@@ -316,6 +320,19 @@ def GET_SITE_URL():
     return site_url
 
 
+def GET_USERCONTENT_URL():
+    return MEDIA_URL
+
+
+def GET_USERCONTENT_QUOTA():
+    try:
+        disk_usage = os.statvfs(MEDIA_ROOT)
+        return disk_usage.f_bavail * disk_usage.f_frsize
```

Comment:
this reads kind of cryptic to me. Can you add e.g. an intermediate variable `availbale_size_in_bytes` or something like that to convey information about what this number is.

Reply:
that is just your code that i stole from the health check :P but yeah can make that better

Reply:
Uhh maybe factor it out as a helper function then 😜

Reply:
Feel free to touch the health check then